README.md

```diff
@@ -9,7 +10,7 @@ Sami2py is a python module that runs the SAMI2 model, as well as archives, loads
 The implementation used here includes several added options to the original release of SAMI2.  A full list is included in ***add link to changes page***, but several of these include:
```

Comment:
Add links for readthedocs

Reply:
Added link.  Should direct to the correct page once merge occurs and docs are built on the master branch.